Heartland-iOS-SDK/Classes/Terminals/HpsTerminalResponse.h

```diff
@@ -84,6 +84,12 @@
 @property (nonatomic,strong) NSString *signatureData;
 @property (nonatomic,strong) NSString *surchargeFee;
 @property (nonatomic,strong) NSString *surchargeAmount;
+@property (nonatomic,strong) NSString *merchantId;
+@property (nonatomic,strong) NSString *accountType;
+@property (nonatomic,strong) NSString *posSequenceNo;
+@property (nonatomic,strong) NSString *currencyCode;
+@property (nonatomic,strong) NSString *pinVerified;
```

Comment:
`pinVerified` is already listed as a property in `HpsUpaResponse`'s header file, please delete it here